src/main.rs

```diff
@@ -126,10 +126,10 @@ impl GameState {
 
 //Main game loop - tweaked from example in GGEZ repo (see https://github.com/ggez/ggez/blob/master/examples/02_hello_world.rs)
 pub fn main() -> GameResult {
-    let cb = ggez::ContextBuilder::new("gamescloset", "cs510");
+    /*let cb = ggez::ContextBuilder::new("gamescloset", "cs510");
     let (ctx, event_loop) = &mut cb.build()?;
 
     let state = &mut GameState::new(ctx)?;
-    event::run(ctx, event_loop, state)
-    //connect4::core::main()
+    event::run(ctx, event_loop, state)*/
+    connect4::core::main()
```

Comment:
I forgot to mention that doing this will run the grid with `cargo run`. Honestly since we aren't really going to create a new game, we may just scrap this file and change the file structure to just have everything from`connect4` (i.e. `core.rs` and `ai.rs` in the `src` folder directly. Can discuss as we approach end of project